explanation/explanation.py

```diff
@@ -1,3 +1,41 @@
-class Explanation():
-    def __init__(self, meta_paths, domain_value):
+from util.datastructures import MetaPath, MetaPathRating
+
+import numpy as np
+
+# mock data
+mock_paths = [MetaPathRating(MetaPath([1, 2], [3]), 3, 1), MetaPathRating(MetaPath([1, 2, 1], [3, 4]), 1, 3),
+              MetaPathRating(MetaPath([1, 3, 2], [1, 1]), 5, 2), MetaPathRating(MetaPath([1, 3, 2], [1, 1]), 4, 7),
+              MetaPathRating(MetaPath([1, 3, 2], [1, 1]), 6, 3)]
+
+# algorithm types
+BASELINE = 'baseline'
+
+
+class Explanation:
+    def __init__(self):
         raise NotImplementedError()
+
+
+class SimilarityScore:
+    meta_paths = None
+    similarity_score = None
+
+    def __init__(self, algorithm_type=BASELINE):
+        self.algorithm_type = algorithm_type
+
+    @staticmethod
+    def fetch_meta_paths(self):
```

Comment:
Is this really static in the final system?

Reply:
Yep, I think so. The resource does not depend on and instance of SimilarityScore